Packs/CommonScripts/Scripts/FeedRelatedIndicators/FeedRelatedIndicators_test.py

```diff
@@ -0,0 +1,70 @@
+                          'score': 0, 'content': "## ['T1345'](https://TLVMAC30YCJG5H:8443/#/indicator/4467):\n ",
+                          'contentFormat': 'markdown', 'timestamp': '2020-08-09T16:33:44.49526+03:00',
+                          'scoreChangeTimestamp': '2020-08-09T16:33:44.49526+03:00', 'isTypedIndicator': False,
+                          'type': 'MITRE ATT&CK', 'context':
+                              {'DBotScore(val.Indicator && val.Indicator == obj.Indicator && val.Vendor && val.Vendor '
+                               '== obj.Vendor)': {'Indicator': 'T1345', 'Score': 0, 'Type': 'MITRE ATT&CK',
+                                                  'Vendor': 'MITRE ATT&CK'},
+                               'MITRE.ATT&CK(val.value && val.value = obj.value)':
+                                   {'customFields': {}, 'indicatorid': '4467', 'value': 'T1345'}}, 'reliability': ''}}},
+     'moduleToFeedMap': {'MITRE ATT&CK': {
+         'reliability': 'A+ - 3rd party enrichment', 'fetchTime': '2020-08-09T16:33:44.49526+03:00',
+         'sourceBrand': 'MITRE ATT&CK', 'sourceInstance': 'MITRE ATT&CK', 'moduleId': 'MITRE ATT&CK',
+         'expirationPolicy': 'indicatorType', 'expirationInterval': 0, 'bypassExclusionList': False, 'score': 0,
+         'classifierVersion': 0, 'classifierId': '', 'mapperVersion': 0, 'mapperId': '', 'type': 'MITRE ATT&CK',
+         'value': 'T1345', 'timestamp': '0001-01-01T00:00:00Z', 'fields': None, 'modifiedTime': '0001-01-01T00:00:00Z',
+         'ExpirationSource': {'setTime': '2020-08-09T16:38:12.845863+03:00', 'source': 'indicatorType', 'user': '',
+                              'brand': 'MITRE ATT&CK', 'instance': 'MITRE ATT&CK', 'moduleId': 'MITRE ATT&CK',
+                              'expirationPolicy': 'indicatorType', 'expirationInterval': 20160}, 'rawJSON': None,
+         'isEnrichment': True}}, 'isShared': False, 'expiration': '2020-08-23T16:38:12.845863+03:00',
+     'manualExpirationTime': '0001-01-01T00:00:00Z', 'expirationStatus': 'active', 'expirationSource':
+         {'setTime': '2020-08-09T16:38:12.845863+03:00', 'source': 'indicatorType', 'user': '', 'brand': 'MITRE ATT&CK',
+          'instance': 'MITRE ATT&CK', 'moduleId': 'MITRE ATT&CK', 'expirationPolicy': 'indicatorType',
+          'expirationInterval': 20160}, 'deletedFeedFetchTime': '0001-01-01T00:00:00Z',
+     'calculatedTime': '2020-08-09T16:38:12.86248+03:00', 'lastReputationRun': '2020-08-09T16:33:44.506101+03:00',
+     'comment': '', 'manuallyEditedFields': None, 'modifiedTime': '2020-08-09T16:33:44.49526+03:00',
+     'aggregatedReliability': ''}], 'searchAfter': [' \x01\x16\x14g#Wjx\x1d\x00', '4467']}
+
+
+def test_main(mocker):
+    from FeedRelatedIndicators import main
+    mocker.patch.object(demisto, 'args', return_value={'indicator': FILE_INDICATOR}),
+    mocker.patch.object(demisto, 'searchIndicators', return_value=SEARCH_INDICAORS_RESPONSE)
+    mocker.patch.object(demisto, 'results')
+    main()
```

Comment:
better to test `feed_related_indicator` instead of `main` and then avoid mocking `results`